frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { expect } from 'chai';
+import {
+  ATTACH_TO_DEPLOYMENT,
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NO_ANNOTATIONS,
+  NOOBAA_LABEL,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  OBC_STORAGE_CLASS_EXACT,
+  SECOND,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    obcHandler = new CreateOBCHandler(OBC_NAME, testName, OBC_STORAGE_CLASS, '');
+    obcHandler.createBucketClaim();
+    cy.exec(`echo '${JSON.stringify(deployment)}' | kubectl create -n ${testName} -f -`);
+    cy.url().then((url) => {
+      obcUrl = url;
+    });
+  });
+
+  beforeEach(() => {
+    cy.wait(10 * SECOND);
+  });
+
+  it('Test if Object Bucket Claim is created sucessfully', () => {
+    cy.byLegacyTestID('resource-title')
+      .should('be.visible')
+      .invoke('text')
+      .should('eq', OBC_NAME);
+    cy.byTestID('resource-status')
+      .should('be.visible', { timeout: 1 * MINUTE })
+      .invoke('text')
+      .should('eq', BOUND);
+  });
+
+  it('Test if owner and creation date are shown correctly', () => {
+    cy.byTestSelector('details-item-value__Owner')
+      .scrollIntoView()
+      .should('exist')
+      .invoke('text')
+      .should('eq', 'No owner');
+    cy.byTestSelector('details-item-value__Created at')
+      .scrollIntoView()
+      .should('exist')
+      .invoke('text')
+      .should('eq', 'a few seconds ago');
+  });
+
+  it('Test if secret data is masked', () => {
+    cy.contains('Reveal Values').should('exist');
+    cy.byTestID('co-copy-to-clipboard__text').as('secrets');
+    cy.get('@secrets').should(($el) => {
```

Comment:
```suggestion
    cy.byTestID('co-copy-to-clipboard__text').should(($el) => {
```
I don't think an alias is needed here.

Reply:
I'm not sure about this but it seems we need to capture the output when working with `byTest___` methods inside a `get` to get the correct type definition, if we need to operate on the passed down element inside the `thenable`. If this is in fact a bug in `Cypress`, it's probably stemming from the `add` wrapper we use to define new commands.

![image](https://user-images.githubusercontent.com/33557095/108382225-95aae880-722e-11eb-9136-7088bafeadd9.png)
![image](https://user-images.githubusercontent.com/33557095/108382242-9c396000-722e-11eb-966c-296db1ff94b8.png)